includes/class-wc-payments-account.php

```diff
@@ -340,18 +340,6 @@ public function maybe_handle_oauth() {
 			return;
 		}
 
-		if ( isset( $_GET['wcpay-connection-success'] ) ) {
```

Comment:
IMO, we don't need this logic anymore since we are passing the return URL while connecting to stripe. Let me know if I am missing anything edge case.